src/supermarket/engines/fieri/Gemfile

```diff
@@ -13,14 +13,12 @@ gemspec
 # To use debugger
 # gem 'debugger'
 group :development, :test do
-  gem "chefstyle", "2.1.3"
+  gem "chefstyle"
   gem "pry"
   gem "rspec-rails", "~> 5.1"
   gem "rubocop-rails", require: false
   gem "sqlite3"
   gem "webmock"
   gem "mixlib-shellout", "~> 3.2"
   gem "mixlib-archive", ">= 0.4", "< 2.0"
-  gem "cookstyle", "7.25.10"
```

Comment:
We cannot remove cookstyle from here .. they are needed as dependencies while rspec

Reply:
ok.. i failed to notice we have added it in gemspec.. in that case this line can be removed. My bad